charts/rucio-daemons/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: rucio-daemons
-version: 1.26.0
+version: 1.26.1
```

Comment:
I just release `1.26.1` so this needs to be `1.26.2` then.

Reply:
Done